cmd/memberagent/main.go

```diff
@@ -192,6 +198,18 @@ func buildHubConfig(hubURL string, useCertificateAuth bool, tlsClientInsecure bo
 			hubConfig.TLSClientConfig.CAData = caData
 		}
 	}
+
+	if header, ok := os.LookupEnv("HUB_KUBE_HEADER"); ok {
```

Comment:
can you add a comment on why we have this?

Reply:
I added comments to explain why we need HUB_KUBE_HEADER header.